src/mail.sh

```diff
@@ -98,6 +134,10 @@ function parse_mail_options()
         options_values["${option^^}"]="sendemail.$option $2"
         shift 2
         ;;
+      --verify | -v)
+        mail_verify
+        shift
```

Comment:
If you call some function inside the parse option, you must exit after that. If that is not what you want, move this function to another place.